base-image/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:15.04
+
+# Install the following utilities (required by openembedded)
```

Comment:
Would be nice to have a link to the documentation where the deps are stated: updating yocto would be easier in this way,